scientific_library/tvb/dsl/LEMS2python.py

```diff
@@ -0,0 +1,68 @@
+from mako.template import Template
+import tvb
+import sys
+import os
+sys.path.append("{}{}".format(os.path.dirname(tvb.__file__),"/dsl/NeuroML/lems"))
```

Comment:
what is happening here? you need to import code from that folder?

Reply:
yes, this is the local altered copy of LEMS